### ISSUE TITLE
The cover image of the draft post is not being removed

### ISSUE THREAD

**Issue by c0d3b0t** [thepracticaldev/dev.to#8240]
**Describe the bug**

Can not remove the cover image of the draft post.

**To Reproduce**

1. Create a draft post with uploaded cover image.
2. Click the "Save draft" button.
3. Click the "Click to edit" link.
4. In the editor click the "Remove" link
5. Click the "Save draft" button
6. See that the cover image has not been removed actually

**Expected behavior**

If I clicked the "Remove" link to remove the cover image, and clicked the save button, I'm expecting to see the cover image removed in the draft post.

**Desktop (please complete the following information):**

- OS: Linux Mint 19.3
- Browser: Google Chrome
- Version:  81.0.4044.138 (Official Build) (64-bit)

**Additional context**

I've managed to delete the cover image by doing an extra step - reloaded the page after clicking the "Remove" link and clicked the "Save draft" button after reload completed.

**Comment by Ridhwana** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Hi @c0d3b0t, I'm going to mark this as a duplicate of https://github.com/thepracticaldev/dev.to/issues/8240

Feel free to add your comment there as well, it'd be helpful!
